File: 2MONTH_LESSON3-master/app/src/main/java/com/example/a2_monthlesson3/MainActivity.java
package com.example.a2_monthlesson3;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    private RecyclerView contactRecyclerView;
    private ArrayList<String> contactList = new ArrayList<>();
    private ContactAdapter adapter = new ContactAdapter(contactList);

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        contactRecyclerView = findViewById(R.id.recycler_view);
        loadData();
        contactRecyclerView.setAdapter(adapter);
    }

    private void loadData() {
        contactList.add("Жылдыз");
        contactList.add("Рахим");
        contactList.add("Бермет");
        contactList.add("Максим");
        contactList.add("Санжар");
        contactList.add("Канбар");
        contactList.add("Тилек");
        contactList.add("Рамис");
        contactList.add("Андрей");
        contactList.add("Алибек");
        contactList.add("Гучи");
        contactList.add("Босс");
    }
}

File: 2MONTH_HW5-master/app/src/main/java/com/example/a2_monthlesson5/MainActivity.kt
package com.example.a2_monthlesson5

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.ProgressBar
import android.content.Intent as Intent1

class MainActivity : AppCompatActivity() {

    lateinit var button: Button
    lateinit var emailEditText: EditText
    lateinit var numberEditText: EditText

    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        button = findViewById(R.id.btnNew)
        emailEditText = findViewById(R.id.et1)
        numberEditText = findViewById(R.id.et2)


        button.setOnClickListener {
            val intent = Intent1(this, SecondActivity::class.java)
            intent.putExtra("email", emailEditText.text.toString())
            intent.putExtra("number", numberEditText.text.toString())
            startActivity(intent)
        }
    }

}